frontend/components/Layout/index.tsx

```diff
@@ -1,15 +1,27 @@
 import { Box, ChakraProvider, theme, VStack } from "@chakra-ui/react";
 import Navbar from "components/Navbar";
 import React, { FC } from "react";
+import FiltersComponent from "components/Filters";
 
 const Layout: FC = ({ children }) => {
   return (
-    <ChakraProvider theme={theme}>
```

Comment:
Why we got rid of the ChakraProvider at all, aren't we going to use some Chakra components?